process/hyperOutportBlocksQueue.go

```diff
@@ -0,0 +1,63 @@
+package process
+
+import (
+	"errors"
+	"sync"
+
+	data "github.com/multiversx/mx-chain-ws-connector-template-go/data/hyperOutportBlocks"
+)
+
+type hyperOutportBlocksQueue struct {
```

Comment:
suggestion to use a mechanism based on buffered channel

Reply:
changed.